test/integration/multinetwork_test.go

```diff
@@ -0,0 +1,295 @@
+	if err != nil {
+		t.Fatalf("failed to delete minikube with args: %q : %v", rr.Command(), err)
+	}
+}
+
+// multinetworkValidateVerifyDeleted makes sure no left over left after deleting a profile such as containers or volumes
+func multinetworkValidateVerifyDeleted(ctx context.Context, t *testing.T, profile string) {
+	defer PostMortemLogs(t, profile)
+
+	rr, err := Run(t, exec.CommandContext(ctx, Target(), "profile", "list", "--output", "json"))
+	if err != nil {
+		t.Errorf("failed to list profiles with json format after it was deleted. args %q: %v", rr.Command(), err)
+	}
+
+	var jsonObject map[string][]map[string]interface{}
+	if err := json.Unmarshal(rr.Stdout.Bytes(), &jsonObject); err != nil {
+		t.Errorf("failed to decode json from profile list: args %q: %v", rr.Command(), err)
+	}
+	validProfiles := jsonObject["valid"]
+	profileExists := false
+	for _, profileObject := range validProfiles {
+		if profileObject["Name"] == profile {
+			profileExists = true
+			break
+		}
+	}
+	if profileExists {
+		t.Errorf("expected the deleted profile %q not to show up in profile list but it does! output: %s . args: %q", profile, rr.Stdout.String(), rr.Command())
+	}
+
+	if KicDriver() {
+		bin := "docker"
+		if PodmanDriver() {
+			bin = "podman"
+		}
+		rr, err := Run(t, exec.CommandContext(ctx, bin, "ps", "-a"))
+		if err == nil && strings.Contains(rr.Output(), profile) {
+			t.Errorf("expected container %q not to exist in output of %s but it does output: %s.", profile, rr.Command(), rr.Output())
+		}
+
+		rr, err = Run(t, exec.CommandContext(ctx, bin, "volume", "inspect", profile))
+		if err == nil {
+			t.Errorf("expected to see error and volume %q to not exist after deletion but got no error and this output: %s", rr.Command(), rr.Output())
+		}
+
+		rr, err = Run(t, exec.CommandContext(ctx, bin, "network", "ls"))
+		if err != nil {
+			t.Errorf("failed to get list of networks: %v", err)
+		}
+		if strings.Contains(rr.Output(), profile) {
+			t.Errorf("expected network %q to not exist after deletion but contained: %s", profile, rr.Output())
+		}
+	}
+
+}
+
+// multinetworkValidateStatus makes sure stopped clusters show up in minikube status correctly
+func multinetworkValidateStatus(ctx context.Context, t *testing.T, profile string) {
+	defer PostMortemLogs(t, profile)
+
+	statusOutput := runStatusCmd(ctx, t, profile, false)
+	var cs cmd.ClusterState
+	if err := json.Unmarshal(statusOutput, &cs); err != nil {
+		t.Fatalf("unmarshalling: %v", err)
+	}
+	// verify the status looks as we expect
+	if cs.StatusCode != cmd.Stopped {
+		t.Fatalf("incorrect status code: %v", cs.StatusCode)
+	}
+	if cs.StatusName != "Stopped" {
+		t.Fatalf("incorrect status name: %v", cs.StatusName)
+	}
+}
+
+// CleanupExtnet removes the external network in docker, no error on failure, used for Cleanup
+func CleanupExtnet(t *testing.T) {
+	ctx2, cancel2 := context.WithTimeout(context.Background(), Seconds(10))
```

Comment:
nit: ctx and cancel are local variables to this function, so no need to have suffix, but that's ok

Reply:
fixed